core/arch/arm/kernel/abort.c

```diff
@@ -568,3 +568,15 @@ void abort_handler(uint32_t abort_type, struct thread_abort_regs *regs)
 		break;
 	}
 }
+
+#ifdef CFG_CORE_WORKAROUND_ARM_NMFI
+void __noreturn itr_core_handler(void)
+{
+	/*
+	 * Note: removed __weak so that when workaround is enabled and if there
```

Comment:
Note: overrides the default implementation of this function so that ...

Reply:
OK